jstz_cli/src/account/mod.rs

```diff
@@ -141,13 +206,18 @@ pub enum Command {
         alias: String,
     },
     /// Lists all accounts
-    List {},
+    List {
```

Comment:
I'd like an alias for this command `ls` for `list`, given we're using convention from the UNIX `ls` command